### ISSUE TITLE
Bid adapters supporting floors but not the floors module

### ISSUE THREAD

**Issue by patmmccann** [prebid/Prebid.js#6465]
## Type of issue
It is expected https://github.com/prebid/Prebid.js/issues/5966 will require bidders accepting the floor to use the getFloor function. 

Here is a quick identification of the adapters missing support: 

`grep floor *BidAdapter.js | grep -v Math.floor | awk '{print $1}' | awk -F ':' '{print $1}' | awk -F '-' '{print $1}' | sort | uniq > ~/floors.txt`

`grep getFloor *BidAdapter.js | awk '{print $1}' | awk -F ':' '{print $1}' | awk -F '-' '{print $1}' | sort | uniq > ~/floorsmod.txt`

`grep ~/floors.txt -f ~/floorsmod.txt --invert-match`

adspendBidAdapter.js @loorke 
adtrueBidAdapter.js @haibau 
advangelistsBidAdapter.js @trchandraprakash 
audiencerunBidAdapter.js @audiencerun 
c1xBidAdapter.js @CathyHuangtw 
cleanmedianetBidAdapter.js @sa1omon 
cosmosBidAdapter.js @dev-cosmos 
dailyhuntBidAdapter.js @montu1996 
ebdrBidAdapter.js @jlzhangdev 
emx_digitalBidAdapter.js @EMXDigital 
fidelityBidAdapter.js @onaydenov 
gamoshiBidAdapter.js @sa1omon 
getintentBidAdapter.js @andreykud 
imonomyBidAdapter.js @imonomy 
improvedigitalBidAdapter.js @jbartek25 
ironsourceBidAdapter.js @lasloche 
komoonaBidAdapter.js @tzafrirb 
kubientBidAdapter.js @onaydenov
lunamediaBidAdapter.js @trchandraprakash 
luponmediaBidAdapter.js @adxpremium 
madvertiseBidAdapter.js @NLopezMad 
mediaforceBidAdapter.js @Niksok 
mgidBidAdapter.js @Gaudeamus 
mobfoxBidAdapter.js @hadarpeer 
mytargetBidAdapter.js @vfedoseev 
nafdigitalBidAdapter.js @vyatsun 
newborntownWebBidAdapter.js @z-sunshine 
nextrollBidAdapter.js @pbrudnick 
orbidderBidAdapter.js @hendrikiseke1979 
otmBidAdapter.js @fedotxxl 
ozoneBidAdapter.js @afsheenb 
platformioBidAdapter.js @varashellov 
revcontentBidAdapter.js @AzizSaleh 
rhythmoneBidAdapter.js @rhythmonebhaines 
riseBidAdapter.js  @lasloche 
rtbdemandBidAdapter.js @rtbdemand 
saambaaBidAdapter.js @trchandraprakash 
sekindoUMBidAdapter.js @nissSK 
smartadserverBidAdapter.js @lowendavid 
smilewantedBidAdapter.js @MaxSmileWanted 
somoBidAdapter.js @travisbeale 
sonobiBidAdapter.js @JonGoSonobi 
sortableBidAdapter.js @shannonAB 
sspBCBidAdapter.js @wojciech-bialy-wpm 
timBidAdapter.js @borisjaskerovich 
tribeosBidAdapter.js @adnanmiljkovic 
truereachBidAdapter.js @mmprebid 
ucfunnelBidAdapter.js @jackhsiehucf 
unicornBidAdapter.js @ctylim 
vrtcalBidAdapter.js @vrtcal-dev 
vubleBidAdapter.js @Roffray 
waardexBidAdapter.js @yegorWaardex 
widespaceBidAdapter.js @mizmaar3 
windtalkerBidAdapter.js @degroat 

In addition, the following bid adapters accept bid.params.reserve, but do not call getFloor

1ad4goodBidAdapter.js @vladgurgov 
adrelevantisBidAdapter.js @ghguo 
bidfluenceBidAdapter.js @francescocristallo 
xhbBidAdapter.js @srinchiera

**Comment by jsnellbaker** [comment]
To clarify - appnexus does not support floors in general at this time.  It's likely that we were flagged because we use `Math.floor()` at a few points in our code.  

May want to double-check if any of these bidders are in the same scenario.

**Comment by patmmccann** [comment]
That's very helpful,  will do

**Comment by patmmccann** [comment]
#6497 solves for one adapter,  thanks @lemmadev !

**Comment by patmmccann** [comment]
@slimkrazy I made an attempt on #6657 but closed it after I ran into some complications.

**Comment by prebidtappx** [comment]
We are working to support floors, we have open pull request #6626

**Comment by patmmccann** [comment]
> We are working to support floors, we have open pull request #6626

Ty, edited to reflect

**Comment by AskRupert-DM** [comment]
Hi - with regards too ozoneBidAdapter.js - the reason you are seeing our adapter flagged up is because we set a key that uses the floor which is totally unrelated to this module.

That said I've started implementing support for the module in my adapter as I have a couple of other bug fixes to address and noticed some issues which I've flagged in this ticket/issue - #6754  (quite possible I am doing something wrong considering the volume of adapters you have supporting this and no one else has appears to flagged the issues I've raised). Will submit an update once I've managed to resolve those issues per #6754.
